README.md

```
@@ -10,6 +15,7 @@
         - [containsTarget](#containstarget)
         - [containsSource](#containssource)
         - [confirmCallback](#confirmcallback)
+        - [optimizeRules](#optimizerules)
```

markdownlint-cli2 (0.20.0)

[warning] Link fragments should be valid

(MD051, link-fragments)